tools/example_scripts/deletemincsqlwrapper.pl

```diff
@@ -1,4 +1,49 @@
 #!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+deletemincsqlqrapper.pl -- This script is a wrapper for deleting multiple MINC
+files at a time and optionally re-inserting them. It will pause for confirmation
+before deleting. B< Projects should modify the query as needed to suit their
+needs>.
+
+=head1 SYNOPSIS
+
+perl tools/example_scripts/deletemincsqlqrapper.pl `[options]`
+
+Available options are:
+
+-profile      : Name of the config file in
+                C<../../dicom-archive/.loris_mri>
+
+-insertminc   : Re-insert the deleted MINC
+
+
+
+=head1 DESCRIPTION
+
+This is an B<example> script that does the following:
+ - Deletes multiple MINC files fitting a common criterion from the database.
+ - Provides the option to re-insert deleted scans with their series UID when
+   using the `-insertminc` flag.
```

Comment:
I am not sure to understand why ` character is used around -insertminc. If they should appear as code, then, should use C<> tag. Also present multiple times in the Notes section.